qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -60,7 +62,22 @@ def __init__(self, experiments: List[BaseExperiment], backend: Optional[Backend]
         super().__init__(experiments, qubits, backend=backend)
 
     def circuits(self):
+        circs = []
+        for circ_group in self._batch_circuits():
+            circs.extend(circ_group[0])
 
+        return circs
+
+    def transpiled_circuits(self):
+        circs = []
+        for circ_group in self._batch_circuits():
```

Comment:
Is it possible to directly call `circ_group[1].transpiled_circuits()`? One may override chid experiment transpile method to inject custom transpile logic, but this will ignore such custom code. I prefer batch experiment to just call child transpile routine as-is, with addition of extra of metadata.

Reply:
It is possible only in the condition that  we add a `layout` parameter to `transpiled_circuits`, because the `initial_layout` transpile option is taken from the root batch experiment.

Reply:
If you call child experiment `transpiled_circuits` method it refers to `circ_group[1].physical_qubits` so we don't need to take care the initial layout -- given we don't do remapping. I think this assumption generally holds in batch execution.